boto3/docs/action.py

```diff
@@ -52,12 +78,18 @@ def document_actions(self, section):
         )
 
         for action_name in sorted(resource_actions):
+            overrides = DOCUMENT_ACTION_OVERRIDES.get(
+                self._resource_name, {}
+            ).get(action_name, {})
             # Create a new DocumentStructure for each action and add contents.
             action_doc = DocumentStructure(action_name, target='html')
             breadcrumb_section = action_doc.add_new_section('breadcrumb')
             breadcrumb_section.style.ref(self._resource_class_name, 'index')
             breadcrumb_section.write(f' / Action / {action_name}')
             action_doc.add_title_section(action_name)
+            warning = overrides.get("warning")
+            if warning:
+                action_doc.add_new_section("warning").write(warning)
```

Comment:
nit: We might want to be more specific than `"warning"` incase we ever need to add an other warning for some reason